README.md

```diff
@@ -45,43 +45,39 @@ You can check out a [blog article](https://www.ably.io/blog/a-scalable-realtime-
 
 1. Clone this repository
 
-   ```sh
-   git clone https://github.com/Srushtika/realtime-quiz-framework.git
-   ```
+```sh
+git clone https://github.com/ably-labs/realtime-quiz-framework.git
+```
 
-2. Change directory into the project folder
+2. Change directory to the project folder and Install dependencies
 
-   ```sh
-   cd realtime-quiz-framework
-   ```
+```sh
+npm install
 
-3. Install the dependencies
+cd realtime-quiz/
```

Comment:
You could add a new script entry in package.json and combine the npm install steps using `&&`. See https://github.com/ably-labs/github-commit-visualizer/blob/baf7a1cb35091bcc063752351512ecc1e8e3e881/package.json#L15

Reply:
that is a nice idea, ATM i am using the legacy set-up instuctions, but sure we should add that `&&` catenation to the readme.